cvmfs/ssl.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_SSL_H_
+#define CVMFS_SSL_H_
+
+#include <string>
+#include <vector>
+
+#include "duplex_curl.h"
+#include "util/posix.h"
+
+#ifdef CVMFS_NAMESPACE_GUARD
+namespace CVMFS_NAMESPACE_GUARD {
+#endif
+
+inline unsigned int count_ssl_certificates(std::string directory) {
```

Comment:
Remove inline but make it a local static `CountSslCertificates`

Reply:
I believe we can discard this now.